pyfr/integrators/writers.py

```diff
@@ -10,6 +10,8 @@
 from pyfr.mpiutil import get_comm_rank_root
 from pyfr.util import rm
 
+import h5py
```

Comment:
Import in the wrong place.  We (with one or two exceptions) go Python imports, other package imports, PyFR imports with there being one line break in between sections.  Import names are sorted alphabetically.  See other PyFR as a reference.